dev/com.ibm.ws.concurrent.persistent_fat_simctrl/build.gradle

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+
+addRequiredLibraries.dependsOn addDerby
+
+dependencies {
+  requiredLibs project(':com.ibm.ws.concurrent.persistent_fat_simctrl')
```

Comment:
this dependency shouldn't be needed -- the FAT bundle gets added to the project classpath by default